patch/.cargo/config.toml

```diff
@@ -0,0 +1,8 @@
+[target.x86_64-pc-windows-msvc]
+rustflags = ["-Ctarget-feature=+crt-static"]
```

Comment:
Leave a comment explaining what this is for (or linking to this bug, although I guess blame does that too).

Reply:
Done